stdlib/gettext.pyi

```diff
@@ -145,8 +146,14 @@ else:
         fallback: bool = False,
         codeset: str | None = None,
     ) -> NullTranslations: ...
+    @overload
+    def install(
+        domain: str, localedir: StrPath | None = None, codeset: None = None, names: Container[str] | None = None
+    ) -> None: ...
+    @overload
+    @deprecated("The `codeset` parameter is deprecated since Python 3.8; removed in Python 3.11.")
     def install(
-        domain: str, localedir: StrPath | None = None, codeset: str | None = None, names: Container[str] | None = None
+        domain: str, localedir: StrPath | None = None, codeset: str = ..., names: Container[str] | None = None
     ) -> None: ...
```

Comment:
```suggestion
    @overload
    @deprecated("The `codeset` parameter is deprecated since Python 3.8; removed in Python 3.11.")
def install(
        domain: str, localedir: StrPath | None, codeset: str, /, names: Container[str] | None = None
) -> None: ...
    @overload
    @deprecated("The `codeset` parameter is deprecated since Python 3.8; removed in Python 3.11.")
def install(
        domain: str, localedir: StrPath | None = None, *, codeset: str, names: Container[str] | None = None
) -> None: ...
```